modules/consensus/consensusdb.go

```diff
@@ -62,6 +69,18 @@ var (
 	SiafundPool = []byte("SiafundPool")
 )
 
+var (
+	// FieldOakInit is a field in BucketOak that gets set to "true" after the
+	// oak initialiation process has completed.
+	FieldOakInit = []byte("OakInit")
+)
+
+var (
+	// ValueOakInit is the value that the oak init field is set to if the oak
+	// difficulty adjustment fields have been correctly intialized.
+	ValueOakInit = []byte("true")
+)
```

Comment:
I am not sure, I don't think the bucket names need to be exported. But if we un-export them, that should happen in another PR. This PR has it because it's consistent.